packages/api/__tests__/specs/mailchimpMailProvider.ts

```diff
@@ -39,10 +50,48 @@ describe('Mailchimp Mail Provider', () => {
     })
 
     expect(mockSend).toHaveBeenCalledTimes(1)
-    expect(mockSend).toHaveBeenCalledWith({
+    expect(mockSend).toHaveBeenCalledWith(
+      {
+        message: {
+          html: undefined,
+          text: 'hello Test',
+          subject: 'test subject',
+          from_email: 'dev@wepublish.ch',
+          to: [
+            {
+              email: 'test@test.com',
+              type: 'to'
+            }
+          ],
+          metadata: {
+            mail_log_id: 'fakeMailLogID'
+          }
+        }
+      },
+      expect.any(Function),
+      expect.any(Function)
+    )
+  })
+})
+
+test('sendMail with template should call mandrill sendTemplate', async () => {
+  await mailChimpMailProvider.sendMail({
+    subject: 'test subject',
+    recipient: 'test@test.com',
+    replyToAddress: 'dev@test.com',
+    mailLogID: 'fakeMailLogID',
+    template: 'test-mail',
+    templateData: {message: 'hello Test'}
+  })
+
+  expect(mockSendTemplate).toHaveBeenCalledTimes(1)
+  expect(mockSendTemplate).toHaveBeenCalledWith(
+    {
+      template_name: 'test-mail',
+      template_content: [],
       message: {
         html: undefined,
-        text: 'hello Test',
+        text: undefined,
         subject: 'test subject',
```

Comment:
Why undefined here ?

Reply:
If i send a mail with a template from Mailchimp i don't need any text. All the text will be in the template from mailchimp